changes/2803-uriyyo.md

```diff
@@ -0,0 +1,2 @@
+Fix issue when `update_forward_refs` fail to resolve `ForwardRef` 
```

Comment:
I think it's easier to have one change file with the PR number with both points inside (key for mappings + normalize)